bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4807,6 +4807,111 @@
+            </sqlCheck>
+        </preConditions>
+        <comment>Add Tablet to Dosing Unit concept set</comment>
+        <sql>
+            SET @set_concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED');
+
+            SET @tablet_concept_id = (SELECT concept_id FROM concept_name where name = 'Tablet' AND concept_name_type = 'FULLY_SPECIFIED');
 
+            INSERT INTO concept_set (concept_id, concept_set,sort_weight,creator,date_created,uuid) values (@tablet_concept_id, @set_concept_id, 1, 1, now(),uuid());
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-202303200340" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(concept_set_id) FROM concept_set WHERE concept_set = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED')
+                AND concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Capsule' AND concept_name_type = 'FULLY_SPECIFIED');
+            </sqlCheck>
+        </preConditions>
+        <comment>Add Capsule to Dosing Unit concept set</comment>
+        <sql>
+            SET @set_concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED');
+
+            SET @capsule_concept_id = (SELECT concept_id FROM concept_name where name = 'Capsule' AND concept_name_type = 'FULLY_SPECIFIED');
+
+            INSERT INTO concept_set (concept_id, concept_set,sort_weight,creator,date_created,uuid) values (@capsule_concept_id, @set_concept_id, 1, 1, now(),uuid());
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-202303061500" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT count(*) FROM concept_name where name='tablet(s)' and concept_name_type='SHORT';
+            </sqlCheck>
+            <sqlCheck expectedResult="1">
+                SELECT count(*) FROM concept_name where name='capsule(s)' and concept_name_type='SHORT';
+            </sqlCheck>
+            <sqlCheck expectedResult="1">
+                SELECT count(*) FROM concept_name where name='unit(s)' and concept_name_type='SHORT';
+            </sqlCheck>
+        </preConditions>
+        <comment>Renaming Concept Names of Dosing Units Type SHORT</comment>
+        <sql>
+            UPDATE concept_name SET name='tablet' where name='tablet(s)' and concept_name_type='SHORT';
+
+            UPDATE concept_name SET name='capsule' where name='capsule(s)' and concept_name_type='SHORT';
+
+            UPDATE concept_name SET name='unit' where name='unit(s)' and concept_name_type='SHORT';
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-202303171600" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
```

Comment:
can we handle updating short name and Fully specified name as part of one liquibase migration

Reply:
@binduak The preconditions differ for both short name updation and insertion of fully specified name. Should we work on the precondition and make it such that it works for both the changes?

Reply:
Updated by handling both the updation of short name and Fully specified name as part of one Liquibase migration.